### ISSUE TITLE
Request for other commands

### ISSUE THREAD

**Issue by TrustyKdawg** [mircearoata/SatisfactoryPakUtilityMod#3]
I appreciate the work you've put in to create a mod that works with the current version of the game replacing the previous utilitymod.
I was hoping that you may be able to add in a couple of commands that were in the previous version that I used constantly. Those were, the "H" shortcut to enable and disable flight as well as the /togglebuildrestrictions.
I have taken a look at trying to do this myself however, I don't know where to start. I hope this might be fairly trivial for you and only take a few minutes to implement?

**Comment by Claysive** [comment]
Yes! Please add the toggle flight button. Such a big QOL function. Also, there used to be a function for setting and locking the time. Would be a great thing too add

**Comment by janissimsons** [comment]
I also would like to set and lock the time, it is a very useful command

**Comment by mircearoata** [comment]
I am making another mod for no build restrictions because that requires c++ and Pak UtilityMod will always be pak only, so that it won't break with updates of the game. Also, it turned out that no build restrictions is more difficult than I expected, there are many functions that need to be overridden

For time there is PermaDay which has different commands to change the time or stop it.
